wand/image.py

```diff
@@ -1028,6 +1110,51 @@ def background_color(self, color):
             if not result:
                 self.raise_exception()
 
+    def matte(self, flag):
```

Comment:
I thought about this twice.  It could be seen as a *write-only field*, but also be seen as an operation.  Don’t mind the above comment too much!